packages/extension/src/newtab/ShortcutLinks/SiteImage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement, SyntheticEvent } from 'react';
+import { WithClassNameProps } from '@dailydotdev/shared/src/components/utilities';
+
+interface SiteImageProps extends WithClassNameProps {
+  url: string;
+  iconSize: number;
+}
+export const SiteImage = ({
+  url,
+  iconSize,
+  className,
+}: SiteImageProps): ReactElement => {
+  const cleanUrl = encodeURIComponent(url);
+  const src = `https://www.google.com/s2/favicons?sz=${iconSize}&domain=${cleanUrl}`;
+  const fallbackSrc = `https://api.daily.dev/icon?url=${cleanUrl}&size=${iconSize}`;
```

Comment:
Should we consider not using it as a fallback to remove this service altogether?

Reply:
Ah the last you mentioned we should use our fallback in case Google randomly stops this endpoint 👍
(Also I see we have a endpoint called `/lettericon` which i'm not sure who uses it? (it's not webapp as far as I can see)

I'm happy to remove it though, and worst case it's broken for a bit.
> Do keep in mind it's extension so would need a deployment to fix it

Reply:
it would be smart to proxy it through API so if it breaks we can point to something else easily without extension republish?

Reply:
Basically it can still be `api.daily.dev/icon` but we now proxy to google instead of icon server deployment

Reply:
Yeah, I like @capJavert's idea, to have API proxy it instead

Reply:
> Ah the last you mentioned we should use our fallback in case Google randomly stops this endpoint

sounds like me 🙈 but let's remove for now.

as for the smart proxy, i have nothing to smart (pun intended) to say. feel free with whatever approach you like

Reply:
It's load balancer proxied at the moment, but since they have different signatures not sure what's the best approach there is? (@omBratteng you might have some ideas here?)

Reply:
@omBratteng can we make a rewrite rule for params on LB or we need to do it on API side?

Reply:
Are we happy with a new endpoint then?
(This time on API not LB)

something like `api.daily.dev/shortcut_icon` it will simply redirect to google, but in case of emergency we can modify it.

Reply:
I think it can even be called the same (we only need to disable one in LB), that will make all extension version automatically work better, even firefox?

Reply:
I'm also fine with that yes, I'll add the route to API. just not sure which one will take priority if both exist?
(probably LB one, which is what we should want)

Reply:
Yap. If we override the exact signature it should work everywhere

Reply:
We could re-claim the /icon for API, and then in API we proxy Google, if not, the iconserver service (`http://iconserver` internally)